### DISCUSSION TITLE
blog/python%E7%BB%98%E5%88%B6%E4%B8%89%E7%BB%B4%E7%90%83

### DISCUSSION THREAD

**Discussion by giscus** [chua-n/chua-n.github.io#151] [Announcements]
# blog/python%E7%BB%98%E5%88%B6%E4%B8%89%E7%BB%B4%E7%90%83

“万恶”的 mayavi 库，琢磨了老半天，才弄清楚怎么根据给定的球心坐标和半径批量绘制出三维空间的球。 1. 思路 mayavi 的 mlab 类似 matplotlib 的 pyplot, 其中points3d函数用来绘制一些三维空间的离散点，在实际情况中，一个点必然需要一种“表示方式”，在points3d函数中，提供了"sphere", "poi...

https://chua-n.com/blog/python%E7%BB%98%E5%88%B6%E4%B8%89%E7%BB%B4%E7%90%83.html

**Comment by chua-n** [comment]
弄哩不赖，中！
